### PR TITLE
Add CI tasks

### LINKED ISSUE #13: Setup CI tasks
Create a Github action that:

* Run the unit tests
* Run a linter
* Build the library

### PR DESCRIPTION
**Pull request by matextrem**
Closes #13 

This PR adds the following CI Tasks: 

- Build
- Test
- Eslint

### CODE REVIEW

**Review by ramirotw — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: 'ESLint check'
+on: [pull_request]
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gnosis/safe-react-eslint-plus-action@v4.0.0
```
Comment (ramirotw) [inline .github/workflows/lint.yml:9]: what's the difference between https://github.com/bradennapier/eslint-plus-action and `gnosis/safe-react-eslint-plus-action`?

Comment (matextrem) [reply .github/workflows/lint.yml:9]: Basically for this [issue](https://github.com/bradennapier/eslint-plus-action/issues/72) in the lib and also last release was about 2 years ago. Gnosis forked the repo and they are keeping it updated since they are using it for the `safe-react` project

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.8.0
+        with:
+          access_token: ${{ github.token }}
+      - name: Remove broken apt repos [Ubuntu]
+        if: ${{ matrix.os }} == 'ubuntu-latest'
+        run: |
+          for apt_file in `grep -lr microsoft /etc/apt/sources.list.d/`; do sudo rm $apt_file; done
```
Comment (anxolin) [inline .github/workflows/build.yml:28]: why is this needed?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (anxolin) [reply .github/workflows/lint.yml:9]: A bit of an irony:
image

**Review by anxolin — APPROVED** [review]
Approving. The workflows felt a bit engineered (with commands, and some third party actions.
